pkg/cmd/datastore/datastore.go

```diff
@@ -279,11 +319,16 @@ func newCRDBDatastore(opts Config) (datastore.Datastore, error) {
 		opts.URI,
 		crdb.GCWindow(opts.GCWindow),
 		crdb.RevisionQuantization(opts.RevisionQuantization),
-		crdb.ConnMaxIdleTime(opts.MaxIdleTime),
-		crdb.ConnMaxLifetime(opts.MaxLifetime),
-		crdb.ConnHealthCheckInterval(opts.HealthCheckPeriod),
-		crdb.MaxOpenConns(opts.MaxOpenConns),
-		crdb.MinOpenConns(opts.MinOpenConns),
+		crdb.MaxOpenReadConns(opts.ReadConnPool.MaxOpenConns),
+		crdb.MinOpenReadConns(opts.ReadConnPool.MinOpenConns),
+		crdb.ReadConnMaxIdleTime(opts.ReadConnPool.MaxIdleTime),
+		crdb.ReadConnMaxLifetime(opts.ReadConnPool.MaxLifetime),
+		crdb.ReadConnHealthCheckInterval(opts.ReadConnPool.HealthCheckInterval),
+		crdb.MaxOpenWriteConns(opts.WriteConnPool.MaxOpenConns),
+		crdb.MinOpenWriteConns(opts.WriteConnPool.MinOpenConns),
```

Comment:
perhaps moving the `Write` world to the beginning of the method name, so this method looks just like the other `Write*` methods? Feels weird a different naming convention was chosen for these.

Reply:
I tried to make it `(Read|Write)Conns` everywhere.

Reply:
right except these 2 methods do not follow the convention